### PR TITLE
Add explanation of argument evaluation. (#170)

### PR DESCRIPTION
**Pull request by thebioengineer**
Hopefully this is clear enough to explain how the arguments evaluate?

We could also address #156 here too if we want?

### CODE REVIEW

File: R/tfrmt.R [hunk 1]
```diff
@@ -23,7 +23,24 @@
...
+#'   tfrmt allows users to pass `vars`, `quo`, and unquoted expressions to a
+#'   variety of arguments, such as `group`, `label`, `param`, `values`,
+#'   `column`, and `sorting_cols`. Users accustomed to tidyverse semantics
+#'   should be familiar with this behavior. However, there is an important
+#'   behavior difference between tfrmt and normal tidyverse functions. Because
+#'   the data are not a part of tfrmt, it does not know when a value being
+#'   passed to it is intended to be an unquoted expression representing a column
+#'   name or an object from the environment. As such, it preferentially uses the
+#'   value from the environment over preserving the entry as an expression. For
+#'   example, if in your environment you have an object "my_object" in your
```
Comment (bzkrouse) [inline R/tfrmt.R:39]:
```suggestion
#'   example, if you have an object "my_object" in your
```

File: R/tfrmt.R [hunk 2]
```diff
@@ -23,7 +23,24 @@
...
+#'   environment with the value "Hello world", and try to create a tfrmt as
+#'   `tfrmt(column = my_object)`, it will take the value of "my_object" over
+#'   assuming the column argument is an unquoted expression and view the entry
+#'   to `column` as "Hello World".
```
Comment (bzkrouse) [inline R/tfrmt.R:43]: should we add a line about how to make sure this doesn't happen (i.e. quoting the input)

Comment (thebioengineer) [reply R/tfrmt.R:43]: That makes sense to add IMO. 
```suggestion
#'   to `column` as "Hello World". To pass "my_object" to tfrmt as a column name, use
#'   quotes around the value: `tfrmt(columnn = "my_object")`.
```

**Review by statasaurus — APPROVED** [review]
Perfect!

**Pull request merged**